python/tvm/relay/frontend/tensorflow.py

```diff
@@ -1204,7 +1208,7 @@ def from_tensorflow(self, graph, layout="NHWC", shape=None, outputs=None):
         return func, self._params
 
 
-def from_tensorflow(graph, layout="NHWC", shape=None, outputs=None):
+def from_tensorflow(graph, layout="NHWC", shape=None, outputs=None, use_dense_op=True):
```

Comment:
I don't think we should have a flag here. We should just commit to one codepath.

Reply:
The problem is that we're not able to remove all the `nn.dense` at this moment and there's not enough AutoTVM template for `nn.matmul`.

So the use of `nn.matmul` can only be seen as a experimental feature. We should not change the default behavior in case this may affect those who are using `nn.dense`.

Reply:
Can't we use the dense schedules when A_transpose=false and B_transpose=true. Then we can convert all nn.dense to nn.matmul.